donate/payments/tasks.py

```diff
@@ -131,6 +169,99 @@ def process_dispute_lost(self, notification):
         })
 
 
+class StripeWebhookProcessor:
+    def process(self, event):
+        # Stripe event types use dot-notation, so convert periods to underscores
+        event_type = event.type.replace('.', '_')
+
+        process_method = getattr(self, 'process_{}'.format(event_type), None)
```

Comment:
`f'process_{event_type}` rather than format? (given the use of formatted strings below, too)